src/index.js

```diff
@@ -131,7 +154,14 @@ function runFireball(options) {
     const debug = options.debug || false;
     const runs = options.runs || 7;
     let count = 0;
-    const fbWorker = new Worker('/fireball-js/fireball_worker.js');
+    var blobURL = URL.createObjectURL(new Blob(
+        ['(', fireballWorker.toString(), ')()'],
+        {type: 'application/javascript'}
+    ));
+
+    const fbWorker = new Worker(blobURL);
+
+    URL.revokeObjectURL(blobURL);
```

Comment:
Yea used it for my beatworker so had the code ready to go. Since I'm using fireball it made sense to update it.